client/client_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"database/sql"
+	"io/ioutil"
+	"net"
+	"strings"
+	"testing"
+
+	"github.com/google/trillian"
+	"github.com/google/trillian/extension/builtin"
+	"github.com/google/trillian/server"
+	"github.com/google/trillian/util"
+	"google.golang.org/grpc"
+)
+
+const (
+	createSQLFile = "../storage/mysql/storage.sql"
```

Comment:
It is. Go always sources relative paths from the same directory the source file is in. 
```
gbelvin$ go test ./client/
ok  	github.com/google/trillian/client	0.261s
gbelvin$ cd client && go test ./
ok  	github.com/google/trillian/client	0.243s
```